routes/user.js

```diff
@@ -13,9 +13,9 @@ const userCtrl = require('../controllers/user');
 
 
 // GET /user
+router.get('/:id', isLoggedIn, userCtrl.show);
 router.get('/', isLoggedIn, userCtrl.index);
-router.post('/', isLoggedIn, userCtrl.create);
-// router.get('/:id', isLoggedIn, userCtrl.show);
+router.post('/post', isLoggedIn, userCtrl.create);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7841248)
```suggestion
router.post("/post", isLoggedIn, userCtrl.create);
```